docs/development.rst

```diff
@@ -429,7 +429,7 @@ For local Docker-based development in maintenance mode:
 You should be good to go!
 
 There is a set of integration tests for maintenance mode. If you'd like to run
-them against your local Docker instance, you must first have done the
+them against your local Docker instance, you will first have to do the
```

Comment:
👍 change to active case. The bullet points may need to be updated now, "load" instead of "loaded", etc. "You will first have to load the sample database, ensure the document has been rendered", etc.